common/src/main/java/io/honeycomb/opentelemetry/DistroMetadata.java

```diff
@@ -22,6 +22,9 @@ public class DistroMetadata {
     public static final String RUNTIME_VERSION_FIELD = "honeycomb.distro.runtime_version";
     public static final String RUNTIME_VERSION_VALUE = System.getProperty("java.runtime.version");
 
+    public static final String OTLP_PROTO_VERSION_HEADER = "x-otlp-version";
+    public static final String OTLP_PROTO_VERSION_VALUE = "0.16.0";
```

Comment:
We'll need to keep this up to date with the underlying OTEL dependency.. Maybe add something to `RELEASING.md`?